SampleApp/Sources/AccessibilityViewController.swift

```diff
@@ -0,0 +1,39 @@
+import BlueprintUI
+import BlueprintUICommonControls
+import UIKit
+
+final class AccessibilityViewController: UIViewController {
```

Comment:
Not a blocker but we might want to add some more accessibility Examples to this view.